main.py

```
@@ -260,27 +278,31 @@ def send_resource_allocation(self, resource_allocation: dict[str, Any]) -> None:
             send_telemetry(self.jwt, telemetry)
         self.switch_state(GameState.PLAY)
 
+    def set_initials(self, initials: str) -> None:
+        if self.player:
+            self.player.name = initials
```

This silently does nothing if `self.player` isn't set.

Isn't that an error that should be brought to the developer's attention?

---

@pydew-j 
I guess I've added this just because in VS Code I have my own configuration of linter, that complains about `self.player` being declared as `Player | None` so it could be None. In fact, at this point, it won't so we could stick to the simple assignment statement without `if`.

---

@pydew-j 
`if` statement has been removed